syntax = "proto3";

import "track/v1/track.proto";

package legumeinfo.microservices.chromosome_service.v1;


service Chromosome {
  rpc Get (ChromosomeGetRequest) returns (ChromosomeGetReply) {}
}


message ChromosomeGetRequest {
  string name = 1;
}


message ChromosomeGetReply {
  legumeinfo.microservices.track.v1.Chromosome chromosome = 1;
}
